content/sensu-go/5.10/reference/datastore.md

```diff
@@ -0,0 +1,163 @@
+  - [Specification](#specification)
+
+## Event storage
+
+Sensu stores the most recent event for each entity and check pair using an embedded etcd (default) or an [external etcd][8] instance.
+You can access event data using the [dashboard][9] events page, [`sensuctl event` commands][10], and the [events API][11].
+To store event data over time, we recommend integrating Sensu with a time series database, like [InfluxDB][12].
+
+## Scaling event storage
+
+**LICENSED TIER**: Unlock enterprise-scale event storage in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][13].
+
+Sensu supports using an external PostgreSQL instance for event storage in place of etcd.
+PostgreSQL can handle significantly higher volumes of Sensu events, letting you scale Sensu beyond etcd's 8GB limit.
+
+Configured with a PostgreSQL event store, Sensu connects to PostgreSQL to store and retrieve event data in place of etcd, while etcd continues to store Sensu configuration data.
+You can access event data stored in PostgreSQL using the same Sensu web UI, API, and sensuctl processes as etcd-stored events.
+
+### Requirements
+
+Sensu supports PostgreSQL 9.5 and later as well as [Amazon Relational Database Service][3] (Amazon RDS).
+See the [PostgreSQL docs][14] to install and configure PostgreSQL.
+
+### Configuration
+
+At the time of enabling the PostgreSQL event store, event data cuts over from etcd to PostgreSQL, resulting in a loss of recent event history.
+No restarts or Sensu backend configuration changes are required to enable the PostgreSQL event store.
+
+_WARNING: At this time, the PostgreSQL event store cannot be disabled once configured._
+
+After installing and configuring PostgreSQL, configure Sensu by creating a `PostgresConfig` resource. See the [specification](#specification) for more information.
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+type: PostgresConfig
+api_version: store/v1
+metadata:
+  name: my-postgres
+spec:
+  dsn: "postgresql://user:secret@host:port/dbname"
+  pool_size: 20
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "type": "PostgresConfig",
+  "api_version": "store/v1",
+  "metadata": {
+    "name": "my-postgres"
+  },
+  "spec": {
+    "dsn": "postgresql://user:secret@host:port/dbname",
+    "pool_size": 20
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+With the `PostgresConfig` resource definition saved to a file (for example: postgres.yml), use sensuctl, [configured as the admin user][1], to activate the PostgreSQL event store.
+
+{{< highlight shell >}}
+sensuctl create --file postgres.yml
+{{< /highlight >}}
+
+At this time, there is no supported method for viewing or deleting a `PostgresConfig` resource.
+To update your Sensu PostgreSQL configuration, repeat the `sensuctl create` process shown above.
+If Sensu cannot connect to a configured PostgreSQL instance, you can expect to see errors in the [Sensu backend logs][2] at the `warn` log level.
```

Comment:
Errors will be written at the `ERROR` log level, and status updates (like for the store being enabled, updated, or disabled) will be issued at `WARN` level.